src/articles/comprehensive-travel-to-elm.md

```diff
@@ -0,0 +1,42 @@
+---
+title: A Comprehensive Travel to Elm
+author: <a href="https://twitter.com/JoGrenat">Jordane Grenat</a>
+date: 2016-12-21T00:00:00.000Z
+draft: true
+---
+
+
+As many of us, I started building my first websites many years ago, using Dreamweaver and a range of shiny background colors. Ugly background colors. But for me it was undoubtedly the best websites ever!
+
+Soon, I wanted to add more interactivity: popup, embedded chat widget, hover effects on images... This was my first discovery of JavaScript, a happy new wonderful world! Except it wasn't. It was awful. I didn't quite understood what I was doing, copying and pasting some code samples, happy when it worked but unable to fix it when it didn't. God, I was bad!
```

Comment:
Nop, c'est une question de rythme. J'ai remplacé par `Except it was not` though.